src/containers/table/Table.js

```diff
@@ -79,7 +79,9 @@ class TableView extends React.Component {
         return a.toLowerCase().localeCompare(b.toLowerCase());
       }),
       selectedTags: [],
-      unSelectedStages: props.stages,
+      unSelectedStages: props.stages.sort((a, b) => {
```

Comment:
no point of sorting stages in alphabetical order, they should have their own order